### PR TITLE
TEL-4770 remove support for sensu

### PR DESCRIPTION
**Pull request by sjpalf**
What did we do?
--

1. Removed support for sensu for checks. Removed the output to Json so now this library can only be used for outputting checks in yaml for grafana. Note that we still output the integrations themselves so that they can be used for sensu custom checks.

References
--

https://jira.tools.tax.service.gov.uk/browse/TEL-4770

Evidence of work
--

1. Passing tests. Tests were all updated so we should have pretty much the same test coverage, but now testing the yaml output rather than Json

Next Steps
--

1. Bump version in alert-config and remove step that calls build to create the Json config file and push to pagerduty_mappings in elasticsearch

Risks
--

1. This is a big change

Collaboration
--

Co-authored-by: Muhammed Ahmed <ma3574@users.noreply.github.com>

### CODE REVIEW

**Review by gavD — APPROVED** [review]
image

File: src/main/scala/uk/gov/hmrc/alertconfig/builder/AlertConfigBuilder.scala [hunk 1]
```diff
@@ -728,35 +538,3 @@ object TeamAlertConfigBuilder {
...
-          val mappedServiceDomain = zoneToServiceDomainMappings.get(zone.get)
-          if (mappedServiceDomain.isEmpty) {
-            logger.error(
-              s"Zone to service domain mapping file '${zoneToServiceMappingFile.getAbsolutePath}' does not contain mapping for zone '${zone.get}'")
-          }
-          mappedServiceDomain
-        case _ => zone
-      }
-
-}
```
Comment (ma3574) [inline src/main/scala/uk/gov/hmrc/alertconfig/builder/AlertConfigBuilder.scala:762]: What did this used to be used for?

File: src/test/scala/uk/gov/hmrc/alertconfig/builder/TeamAlertConfigBuilderSpec.scala [hunk 2]
```diff
@@ -19,8 +19,25 @@ package uk.gov.hmrc.alertconfig.builder
...
+  YamlHttp90PercentileResponseTimeThresholdAlert,
+  YamlHttpAbsolutePercentSplitDownstreamHodThresholdAlert,
+  YamlHttpAbsolutePercentSplitDownstreamServiceThresholdAlert,
+  YamlHttpAbsolutePercentSplitThresholdAlert,
+  YamlHttpStatusPercentThresholdAlert,
+  YamlHttpStatusThresholdAlert,
+  YamlHttpTrafficThresholdAlert,
+  YamlLogMessageThresholdAlert,
+  YamlTotalHttpRequestThresholdAlert
+}
```
Comment (ma3574) [inline src/test/scala/uk/gov/hmrc/alertconfig/builder/TeamAlertConfigBuilderSpec.scala:40]: Could just become:
```
import uk.gov.hmrc.alertconfig.builder.yaml._
```

**Review by ma3574 — COMMENTED** [review]
Looks good

**Pull request merged**